client/ayon_hiero/api/plugin.py

```diff
@@ -860,15 +859,18 @@ def _convert_to_tag_data(self):
             # adding tag metadata from ui
             for _key, _value in self.pre_create_data.items():
                 if _key in self.tag_keys:
+                    # backward compatibility for reviewTrack (2024.11.07)
+                    if "reviewTrack" in _key:
```

Comment:
This probably should not override `"reviewableSource"` if is already there.
```suggestion
                    if (
                        "reviewTrack" in _key
                        and "reviewableSource" not in self.tag_data
                    ):
```

Reply:
fixed in 8cbf30c2b1e81d73c2b5a8bf473e1bd5e39800a5